spoonbill_web/settings/settings.py

```diff
@@ -216,7 +216,7 @@
 
 DATAREGISTRY_JAIL = bool(strtobool(os.getenv("DATAREGISTRY_JAIL", "True")))
 
-DATAREGISTRY_MEDIA_ROOT = os.getenv("DATAREGISTRY_MEDIA_ROOT", None)
+DATAREGISTRY_MEDIA_ROOT = os.getenv("DATAREGISTRY_MEDIA_ROOT", "data/exporter")
```

Comment:
I think missing initial `/`? Can you check if they use `os.path.join` (rather than `+`, in which case we need to add `/` at the end)?

```suggestion
DATAREGISTRY_MEDIA_ROOT = os.getenv("DATAREGISTRY_MEDIA_ROOT", "/data/exporter")
```

Reply:
It actually should match the data registry dir /data/storage/exporter_dumps, and it is safe to not use / at the end, I will change it

Reply:
Hmm, I don't like coupling it to the current directory on the host.

This is similar to Collect and Process, back when Collect was sending full paths. We changed it so that Collect sends relative paths, so that we can just configure the host directory in each app separately.

Right now, instead of changing Spoonbill, we set `EXPORTER_HOST_DIR` in data-registry. Instead, we can rename it to `SPOONBILL_EXPORTER_DIR`, to be from Spoonbill's perspective rather than the host's.

Reply:
> We changed it so that Collect sends relative paths, so that we can just configure the host directory in each app separately.

I think this is the best approach and we should replicate it in registry and spoonbill in the future too. But yes, I can change `EXPORTER_HOST_DIR` to `/data/exporter` for now

Reply:
Hmm actually, we are using `EXPORTER_HOST_DIR` in the registry's pillar to point to the host dir so then we can mount that dir in /data/exporter in the registry's container see https://github.com/open-contracting/deploy/blob/9726312b91929fe237de9b96fdd946f0ae7b8983/salt/docker_apps/files/registry.yaml#L16. So I guess we cannot change that one to /data/exporter.

Reply:
Maybe it is easier/clearer if I just change both apps to send and receive relative paths?

Reply:
In the Docker Compose file, we only used `EXPORTER_HOST_DIR` to avoid repetition, but since `EXPORTER_HOST_DIR` will need to be different now,  eitherthe host directory can be set explicitly, or we can use some other Pillar key (only if it's repeated).

Can you create the issue about using relative paths in spoonbill-web?

Reply:
https://github.com/open-contracting/spoonbill-web/issues/405

I will update the deploy PR, the value is repeated indeed, for setting the registry's volume and then the spoonbill one. I guess we could rename `EXPORTER_HOST_DIR` to `SPOONBILL_EXPORTER_DIR` in the registry and then use `EXPORTER_HOST_DIR` only in pillar to set the host dir. But not sure if that is worth it as we are going to change the registry and spoonbill for use relative paths anyway. So maybe we can just repeat the values for now